content/departments/engineering/product/process/index.md

```diff
@@ -38,6 +38,30 @@ See [strategy](../../../../strategy-goals/index.md).
 - [Pricing and tiering/packaging](gtm/pricing.md)
 - [Licensing](gtm/licensing.md)
 
+#### Experimenting on sourcegraph.com
+
+We welcome experimentations on sourcegraph.com. It should be very easy to experiment on .com, send a tweet or email campaign out to get some traffics, and get early feedback in.
+
+The bar for an experiment on .com is the following:
```

Comment:
Do we want to say all experiments must be all of these bars or do we want to say that experiments that do not meet these bars may be subject to extra approval (with a note about how to get that).

For example, I could imagine that we allow an experiment which changes an existing UI or tweaks a main flow but which is small enough risk that we're okay with that as long as the right folks sign off on it.

Reply:
Yup, sounds good.
Who should approve? I'm guessing it should be you? I'll merge without this, sounds like this could be a follow up PR?

Reply:
Maybe start with #ask-product so we don't have a single bottleneck? And then we can pull in additional folks as necessary (e.g., some things might just be a product call, some might require pulling in legal).